addons/vehicles/initSettings.sqf

```diff
@@ -0,0 +1,10 @@
+
+[
+    QGVAR(keepEngineRunning),
+    "CHECKBOX",
+    [LSTRING(SettingKeepEngineRunningName), LSTRING(SettingKeepEngineRunningDesc)],
+    localize ELSTRING(common,ACEKeybindCategoryVehicles),
+    false, // default value
+    true, // isGlobal
+    {[QGVAR(keepEngineRunning), _this] call EFUNC(common,cbaSettings_settingChanged)}
```

Comment:
This does not add the actions when off at mission start and then turned on. `cbaSettings_settingChanged` not correctly implemented I think.

Reply:
Yes, it will not. Question is, should it be changeable mid mission?

Reply:
Whenever it is possible settings should be live editable and take effect immediately.